test/json-forms.test.ts

```diff
@@ -23,17 +22,17 @@ test('jsonforms only data', t => {
     t.deepEqual(jsonForms.dataSchema, jsonSchema);
     t.deepEqual(jsonForms.uiSchema, generateDefaultUISchema(jsonSchema));
 });
-test('jsonforms data and schema no render', t => {
-    const jsonForms = new JsonForms();
+test('Connect JSON Forms element with data and data schema set', t => {
+    const jsonForms = new JsonFormsElement();
     jsonForms.data = {name: 'foo'};
     const dataSchema = {type: 'object', properties: {name: {type: 'string'}}} as JsonSchema;
     jsonForms.dataSchema = dataSchema;
     jsonForms.connectedCallback();
     t.is(jsonForms.children.length, 0);
 });
-test.cb('jsonforms data and schema', t => {
+test.cb('Connect JSON Forms element asynchronously with data and data schema set', t => {
```

Comment:
the async comes from the dataschema resolution, so it is just needed to run the test successfully